### ISSUE TITLE
multiple-thread download 

### ISSUE THREAD

**Issue by dstsmallbird** [astrada/google-drive-ocamlfuse#724]
Hi, is there a way to enable multiple-thread downloading?

I need to read some in-sequence data from Drive while the target is too large to cache.

Thanks.

**Comment by astrada** [comment]
If you don't need to modify those files, you can enable streaming (with `stream_large_files=true`).

**Comment by dstsmallbird** [comment]
> If you don't need to modify those files, you can enable streaming (with `stream_large_files=true`).

Yes, I have enabled this option. But the download speed is pretty slow. If possible, I want to use multiple-thread downloading.

**Comment by digi24** [comment]
> If you don't need to modify those files, you can enable streaming (with `stream_large_files=true`).

I came across a similar situation.  As far as I understand `stream_large_files` applies to the read ahead, so pre-fetsching files sequentially, but not simultaneosly reading a file at different offsets.
